img/i/icedrive.net.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="800" height="800" viewBox="0 0 1024 1024"><circle cx="512" cy="512" r="512" style="fill:#1f4fd0"/><path d="M476.88 258.41C532 250.77 589.62 261.39 638 289c54.28 30.53 97 81.45 116.52 140.69 16.71 48.8 17.87 102.62 4 152.26a256.48 256.48 0 0 1-96 137.42c-42.36 31-94.6 48.48-147.1 48.59a252.5 252.5 0 0 1-157.68-51.19c-50.49-37.72-86-94.87-97.39-156.9-11.4-60.07-.78-124.17 30.4-176.86 38.62-67.18 109.25-114.78 186.13-124.6m-76.49 135.95c-20.82-11.19-48.29 2.33-53.85 24.87-14.89 9.82-29.46 20.5-41.72 33.51-14.86 49.8-10.3 105 12.21 151.83C340.53 654.71 384.18 695 436 714.51a215.88 215.88 0 0 0 121.46 9.23 96.58 96.58 0 0 1-12.15-22.89c-14.38-38.71 3.78-87 41.77-104.39 19-11.21 41.77-3.89 62.19-9.39 26-6.25 47.7-25 61.65-47.3 5.37-7.62.86-17.41-4.88-23.37-23-23-53.82-35.25-82.42-49.34-11.78-14.41-13-37.05-29.89-47.73-56.26-37-129.37-44.72-193.3-25m127.89 59.32c-6.17 3.38-13.84 11.27-7.81 18.32 15 0 30 1.74 45 .38 6-1.5 13.15-8.08 8.37-14.38-6.2-2.82-13.09-3.41-19.61-5.18-8.42-2.15-18.06-4.05-25.94.86" style="fill:#fff"/></svg>
```

Comment:
How about this one instead:
![icedrive net](https://user-images.githubusercontent.com/520841/235480072-0f8de038-58b8-4e1b-811a-84799a0e7f7f.svg)
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 84.7 84.7"><defs><linearGradient id="a" x1="87.7" x2="107.3" y1="224.3" y2="204.7" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#1255ba"/><stop offset="1" stop-color="#06d2d1"/></linearGradient><linearGradient id="b" x1="87.4" x2="101.9" y1="226.4" y2="226.4" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#fff"/><stop offset="1" stop-color="#9dd8f2"/></linearGradient><linearGradient id="c" x1="83" x2="90.5" y1="211.9" y2="211.9" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#fff"/><stop offset="1" stop-color="#9dd8f2"/></linearGradient></defs><g transform="matrix(2.65 0 0 2.65 -211.7 -529.2)"><circle cx="96" cy="216" r="16" fill="#fff" class="B"/><path fill="url(#a)" d="M89 208.7c-.4 0-3.1 1.4-3.3 1.5l-1.3.9c-.7.5-2 1.8-2.6 2.6a16.6 16.6 0 0 0-1.5 2.4l-.2.5v-.6a16 16 0 0 1 4.7-11.4 16 16 0 0 1 27.1 12.8 16 16 0 0 1-6 11 16.3 16.3 0 0 1-10.7 3.5s.8-11 4.6-11c4.6-.1 4.5-.1 5.5-.5s2.2-1.4 3-2.4c.3-.5.3-.8.1-1.3-.3-.7-1.8-1.8-3.9-2.8l-1.3-.6c-.4-.2-.3 0-1-1.7-.5-.9-.7-1.2-1.4-1.6a14.6 14.6 0 0 0-11.3-1.5z" class="B"/><path fill="url(#b)" d="M97.9 226a5.4 5.4 0 0 1 4-5.2h-2l-5 .2a9.6 9.6 0 0 0-3.7 1.4c-.8.6-2 1.8-2.6 2.6a11 11 0 0 0-.8 1.4c-2.5 7.1 7.3 5.4 7.4 5.5H98a15.5 15.5 0 0 0 3.4-.9 5.4 5.4 0 0 1-3.5-5z" class="B"/><path fill="url(#c)" d="M85.5 210.7v-.4l-1 .8a10.2 10.2 0 0 0-.9.7 3.7 3.7 0 0 0 5.6.9 2.4 2.4 0 0 1-3.7-2z" class="B"/><path fill="#2296e2" d="M97.2 212.2c-.6.3-1 .8-.7 1l1.3.2 1.4-.1c.4-.2.6-.4.5-.6 0-.2-.1-.2-.8-.4-.8-.2-1.4-.3-1.7-.1z" class="B"/><circle cx="87.9" cy="210.7" r="2.4" fill="#fff" class="B"/></g></svg>
```